README.md

```diff
@@ -12,91 +12,10 @@ Our goal is to build a generalizable monitoring and evaluation (M&E) platform th
-|            [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/hussain-ali-58a9b690/)             |         [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/rubenandresramirez)         |                [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/bryan-diaz-816a081b9/)                |                 [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/jcroblesorozco/)                |
-
-
-<br>
-
-### **Back-End Team (Labs 33)**
-|                                                   [Chris Sutton](https://github.com/suttonchristopher)                                                       |                                                      [Brandon O'Neal](https://github.com/brandononeal)                                                       |
-| :----------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------: |
-| [<img src="https://res.cloudinary.com/water-my-plants/image/upload/v1619474173/ChrisS.jpg" width = "200" />](https://github.com/suttonchristopher) | [<img src="https://res.cloudinary.com/water-my-plants/image/upload/v1619473829/BrandonON.jpg" width = "200" />](https://github.com/brandononeal) |
-|                                  [<img src="https://github.com/favicon.ico" width="32"> ](https://github.com/suttonchristopher)                                   |                               [<img src="https://github.com/favicon.ico" width="32"> ](https://github.com/brandononeal)                          |
-|            [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/suttonchristopher/)             |         [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="32"> ](https://www.linkedin.com/in/brandonaoneal)         |
-
-<br>
-<br>
-
-![https://mit-license.org](https://img.shields.io/packagist/l/doctrine/orm.svg)
-![https://reactjs.org](https://img.shields.io/badge/react-v16.13.1-blue)
-![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)
-
-- Fork and clone the repo to install it as your own remote.
-  - **note** please [be sure to set your remote](https://help.github.jp/enterprise/2.11/user/articles/changing-a-remote-s-url/) for this repo to point to your Labs Team Front End Repository.
-- run: `npm install` to download all dependencies.
-- run: `npm start` to start your local development server.
-
-> When using Okta for authentication, the app will need to run locally on port 3000.
-
-### Key Features
```

Comment:
Might be a good idea to cut it off here, leaving key features and what's below but removing the clutter from above